Controller/SettingsController.php

```diff
@@ -17,22 +15,22 @@ class SettingsController extends Controller {
 
 	public function modifyAction(Request $request) {
 		$em = $this->getDoctrine()->getManager();
-		$repo = $em->getRepository(get_class(new Setting()));
+		$repo = $em->getRepository('Craue\ConfigBundle\Entity\Setting');
 		$allStoredSettings = $repo->findAll();
 
 		$formData = array(
 			'settings' => $allStoredSettings,
 		);
 
-		$form = Kernel::VERSION_ID < 20800
-			? $this->createForm(new LegacyModifySettingsForm(), $formData)
-			: $this->get('form.factory')->createNamed('craue_config_modifySettings', 'Craue\ConfigBundle\Form\ModifySettingsForm', $formData);
+		$form = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
+			? $this->get('form.factory')->createNamed('craue_config_modifySettings', 'Craue\ConfigBundle\Form\ModifySettingsForm', $formData)
```

Comment:
Then I might've hit a bug here, but tests fail if I change this line to `? $this->createForm('Craue\ConfigBundle\Form\ModifySettingsForm', $formData)` while `getBlockPrefix` returns `craue_config_modifySettings` as well since `modify_settings_form` is actually used.